lms/djangoapps/teams/tests/test_models.py

```diff
@@ -238,7 +238,7 @@ def test_get_memberships(self, username, course_ids, team_ids, expected_count):
     def test_user_in_team_for_course(self, username, course_id, expected_value):
         user = getattr(self, username)
         self.assertEqual(
-            CourseTeamMembership.user_in_team_for_course(user, course_id),
+            CourseTeamMembership.user_in_team_for_teamset(user, course_id),
```

Comment:
This test can just go at this point, I believe.

Reply:
yeah I'd vote removing this test and making the argument required.

Reply:
Removed deprecated test.